apps/architect-web/src/styles/tailwind.css

```
@@ -674,49 +689,21 @@
-	.tippy-box[data-theme~="error"] .tippy-arrow {
-		color: var(--color-error);
+/* A4-sized container for the protocol summary print view. Sets page
+   dimensions and base font size as CSS variables (consumed by descendant
+   .page-break-marker "paper cards"), plus print-mode overrides so the
+   printed output renders in a system font stack independent of the app's
+   webfonts and with full-bleed colors. */
+@utility protocol-summary-surface {
+	--page-size-height: 29.7cm;
+	--page-size-width: 21cm;
+	--base-font-size: 12px;
+	font-size: var(--base-font-size);
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)